spec/rspec/core/memoized_helpers_spec.rb

```diff
@@ -363,6 +363,97 @@ def not_ok?; false; end
         expect(subject).to eq(3)
       end
     end
+
+    describe 'threadsafety', :threadsafe => true do
+      # Force call to eq to happen in threaded env to prevent "circular require" warnings
+      # Not actually sure why calling eq in two threads at once causes this (it's a sporadic failure):
```

Comment:
Oh, nice :) Didn't realize they did that, I'd heard they were going to remove it, but making it threadsafe seems better.